en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/defining-a-backend-security-scheme.md

```diff
@@ -20,27 +20,38 @@ When an actual backend service of the API is protected using basic authenticatio
         username: "admin"
         password: "admin"
   ```
+
 ## Invoking an API using basic authentication
 
-When running the Choreo Connect, you can provide the username, password as adapter environment variables at `docker-composer.yaml` file.
+When running the Choreo Connect, you can provide the username, password as Adapter environment variables in the following configurations files based on the deployment that you are using.
+
+| **Deployment** | **Configuration file** |
+| Docker | `docker-composer.yaml` file. |
+| Kubernetes | `adapter-deployment.yaml` file. |
+
 
 !!! note 
-    If a username/password is not specified as environment variables, the username, password defined in the API Definition section will be used while deploying the 
-    API to Choreo Connect.
+    If a username/password is not specified as environment variables, the username, password defined in the API Definition section will be used while deploying the API to Choreo Connect.
 
 ``` java tab="Format"
 <API-ID>_<endpoint-type>_basic_username=<username>
 <API-ID>_<endpoint-type>_basic_password=<password>
 ```
 
-``` java tab="Example"
+``` java tab="Docker Example"
   services:
       adapter:
           environment:
           - api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username="admin"
           - api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_password="admin"
 ```
 
+``` java tab="Kubernetes Example"
+
+TODO
```

Comment:
@pubudu538 , please add the correct K8s code snippet

Reply:
```suggestion
containers:
        - name: choreo-connect-adapter
          env:
            - name: api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username
              value: "admin"
            - name: api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_password
              value: "admin"
```

Reply:
```suggestion
containers:
    - name: choreo-connect-adapter
      env:
        - name: api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username
          value: "admin"
        - name: api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_password
          value: "admin"
```